src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -209,7 +209,18 @@ MOBILE_INTEGRATION_TEST_TOKEN = '{{ MOBILE_INTEGRATION_TEST_TOKEN }}'
 
 ####### For Constructing Global URLs ########
 
-LOCAL_APPS = ('dimagi.utils', 'gunicorn',)
+LOCAL_APPS = (
+    'dimagi.utils',
+    'gunicorn',
+    {% for app_name, host in localsettings.get('LOCAL_APPS', {}).items() %}
```

Comment:
Is this referencing the same `LOCAL_APPS` that was added in *environments/production/public.yml*? If yes, the structure of that doesn't seem to match what you're doing here. It looks like you'll get the following values on the first and only iteration:
```py
app_name = 'apps'
host = {'name': 'ddtrace.contrib.django', 'host': 'webworkers'}
```
I must be missing something.

Reply:
I got some suggestions from emord and am currently working on overhauling this logic! I will tag you when it's ready to go again

Reply:
@millerdev this is ready again!